test/HTTP2/ServerSpec.hs

```diff
@@ -158,8 +159,18 @@ runClient allocConfig =
     runHTTP2Client s = E.bracket (allocConfig s 4096)
                                  freeSimpleConfig
                                  (\conf -> C.run cliconf conf client)
-    client sendRequest = mapConcurrently_ ($ sendRequest) clients
-    clients = [client0,client1,client2,client3,client3',client3'',client4,client5]
+
+    client :: C.Client ()
+    client sendRequest = mapConcurrently_ id $ [
```

Comment:
I think we can use `foldr1 concurrently_` instead of `mapConcurrently_ id $ `.

Reply:
Sure, that would be equivalent.